lib/components/artwork_grids/infinite_scroll_grid.js

```diff
@@ -280,3 +298,33 @@ export default Relay.createContainer(InfiniteScrollArtworksGrid, {
     `,
   },
 })
+
+const GeneInfiniteScrollContainer = Relay.createContainer(InfiniteScrollArtworksGrid, {
+  initialVariables: {
+    totalSize: PageSize,
+    medium: '*',
+    priceRange: '*-*',
+    sort: '-partner_updated_at',
+  },
+  fragments: {
+    gene: () => Relay.QL`
+      fragment on Gene {
+        artworks: artworks_connection(sort: $sort, price_range: $priceRange, medium: $medium, first: $totalSize, for_sale: true){
+          pageInfo {
+            hasNextPage
+          }
+          edges {
+            node {
+              image {
+                id
+              }
+              ${Artwork.getFragment('artwork')}
+            }
+          }
+        }
+      }
+    `,
+  },
+})
+
+export { GeneInfiniteScrollContainer }
```

Comment:
In retrospect, I think it might be cleaner to move the basic grid layout work into a dumb component and have dedicated components for artist and gene artwork grids that do the fetching.

Also not a blocker, though.